### PR TITLE
Update dotfiles input help text design

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

Following the changes in https://github.com/gitpod-io/gitpod/pull/7337, this will update dotfiles input help text design to match the design of other help text instances accross the product.

## How to test
1. Go to [`/preferences`](https://gitpod.io/preferences)
2. Notice how the dotfiles input help text matches the design of other help text instances accross the product. For example, check the help text when adding a new environment variable in [`/variables`](https://gitpod.io/variables).

| BEFORE | AFTER |
|-|-|
| dotfiles-before | dotfiles-after |

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@gtsiolis: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **gtsiolis** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
@jankeromnes could you take a look at this minor change when you're free?

File: components/dashboard/src/settings/Preferences.tsx
Comment (jankeromnes) [inline components/dashboard/src/settings/Preferences.tsx:193]: Nit: You could add the `mt-1` class to the `<p/>` element below, in order to avoid extra nesting. Not very important though.

**Review by jankeromnes — APPROVED** [review]
Nice! 🎯 Very subtle yet remarkable improvement ✨

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: b1364a7b456d86e1a4ccdc4b498d641ea2edeac3</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7493#pullrequestreview-846396349" title="Approved">jankeromnes</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jankeromnes** [comment]
Okay fine @roboquat 🙄 

/approve no-issue

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7493#issuecomment-1007238690" title="Approved">jankeromnes</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7493#issuecomment-1007238690" title="Approved">jankeromnes</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/dashboard/src/settings/Preferences.tsx
Comment (gtsiolis) [reply components/dashboard/src/settings/Preferences.tsx:193]: Tried to replicate the semantics structure we already use elsewhere:

https://github.com/gitpod-io/gitpod/blob/e2c1988ec60a4c73875850ffd7d688bfc230dff8/components/dashboard/src/settings/EnvironmentVariables.tsx#L73-L75

However, I agree we could make this could be improved. Creating a component for this form element could help. 💡

**Comment by gtsiolis** [comment]
Thanks, @jankeromnes! 🏀

**Pull request merged**
